pandas/_libs/parsers.pyx

```diff
@@ -845,10 +842,9 @@ cdef class TextReader:
 
             passed_count = len(header[0])
 
-            # if passed_count > field_count:
-            #     raise ParserError('Column names have %d fields, '
-            #                        'data has %d fields'
-            #                        % (passed_count, field_count))
```

Comment:
is this related to the rest of the PR?  we definitely dont want commented-out code lying around, but it isnt obvious to me that enabling it is the right thing to do

Reply:
Gotta be honest, It just bothered me that when I run grep ```parsers.pyx``` is popping because of this.

I don't mind changing it back, if that really matters.

Reply:
I completely understand.  If you grep for `# FIXME:` you'll find a bunch of "don't leave commented-out code" that I put in whenever I come across it and don't immediately know what to do with it.

Reply:
so for the time being leaving it with the change?
or without?

Reply:
pls revert and either a) add a FIXME comment or b) re-enable or delete in a separate PR

Reply:
Will delete in a separate pr.
I think that this deserves a discussion on its own.